drivers/gpio/gpio_xlnx_axi.c

```diff
@@ -281,7 +281,7 @@ static const struct gpio_driver_api gpio_xlnx_axi_driver_api = {
 		.all_outputs = DT_INST_PROP_OR(n, xlnx_all_outputs2, 0),\
 	};								\
 									\
-	DEVICE_AND_API_INIT(gpio_xlnx_axi_##n##_2,			\
+	DEVICE_DT_INST_DEFINE(gpio_xlnx_axi_##n##_2,			\
```

Comment:
This doesn't look right.  Is the first argument a node id?  Where's the pm control?  Same below.

Reply:
NAK to make sure this is addressed.

Reply:
Fixed.